quartz/components/ContentMeta.tsx

```diff
@@ -38,7 +41,15 @@ export default ((opts?: Partial<ContentMetaOptions>) => {
         segments.push(displayedTime)
       }
 
-      return <p class={classNames(displayClass, "content-meta")}>{segments.join(", ")}</p>
+      const segmentsElements = segments.map((segment, index) => (
+        // Check if the current segment is not the last one to append the comma
+        <span key={index}>
+          {segment}
+          {index < segments.length - 1 ? (options.showComma ? "," : "") : ""}
+        </span>
+      ))
```

Comment:
let's just do this through css

```suggestion
      const segmentsElements = segments.map((segment) => (
        // Check if the current segment is not the last one to append the comma
        <span show-comma={options.showComma}>{segment}</span>
      ))
```

Then we can use &::after for this. @jackyzha0 any opinions with this approach instead?

```sass
.content-meta {
	span {
		margin-right: 8px;
    
    [data-show-comma] {
      &::after {content: ","}
    }
	}
}

Reply:
I'm all for this approach but you still need the index calc to make sure the last segment doesn't get a comma 
<img width="184" alt="image" src="https://github.com/jackyzha0/quartz/assets/4028391/80e94206-8a8d-4e8e-a614-8d869edf4912">

Something a little more complex like this could work

```
<span key={index} data-show-comma={index < segments.length - 1 && options.showComma ? 'true' : undefined}>
          {segment}
</span>
```

Reply:
can we just use `:nth-last-child` or `:last-of-type` selector or sth similar iirc?

Reply:
I always forget about those selectors, just made the change